dev/com.ibm.ws.microprofile.rest.client_fat/publish/servers/mpRestClient11.produceConsume/server.xml

```diff
@@ -3,6 +3,7 @@
     <feature>componenttest-1.0</feature>
     <feature>mpRestClient-1.1</feature>
     <feature>ssl-1.0</feature>
+    <feature>servlet-4.0</feature>
```

Comment:
```suggestion
    <feature>servlet-3.1</feature>
```
Changing to 4.0 will change the test to run EE8.  I assume we want to keep the test running the same that it was.

Reply:
Hmm, that's unfortunate. The test configuration has the first repeat as `MP20`, which would be EE8 features. If we want this test to run on EE7, we should update the first repeat to use `MP14`.